scalac-scoverage-runtime/shared/src/main/scala/scoverage/Invoker.scala

```diff
@@ -67,6 +148,39 @@ object Invoker {
     override def accept(pathname: File): Boolean = pathname.getName.startsWith(MeasurementsPrefix)
   })
 
+
+  def resetDataDir(dataDir: String): Unit = {
+    // Making the directory if it does not exist
+    new File(dataDir).mkdirs()
```

Comment:
Is this a safe operation?

Reply:
It returns true if the directories are made and false otherwise (in case the directories are already present). It raises an exception only if a SecurityManager exists and denies to create the named dir. https://www.tutorialspoint.com/java/io/file_mkdirs.htm